app/main.py

```diff
@@ -1,3 +1,20 @@
 def cache(func):
-    # Write your code here
-    pass
+
+    cached_list = {}
+
+    def inner(*args):
+
+        for key, value in cached_list.items():
+            if key == args:
+
+                cached_list[args] = "Getting from cache"
+                print(cached_list[args])
+
+                for item in args:
+                    return item
+
+        cached_list[args] = "Calculating new result"
+        print(cached_list[args])
+
+        return func(*args)
```

Comment:
you should return the result from the cache

Reply:
When I try to return the result from the cache I get an error ( function missing 2 required positional arguments: 'b' and 'c'). How do I fix this?